perf/lua/1mops_write.lua

```diff
@@ -67,7 +80,7 @@ if params.help or params.h then
 end
 
 -- turn true to test the qsync
-local test_qsync = params.qsync or false
+local test_qsync = params.sync or false
```

Comment:
Looks like a mess: in comment `qsync` and variable is `test_qsync`, but `params.sync`. Should we rename `qsync` to `sync` everywhere?

Reply:
donw

Reply:
Thanks!